### PR TITLE
Grammatical fixes delegated to: dsbrown1331 by JustinHart

### PR DESCRIPTION
**Pull request by justinhart**
Fixes for English usage to the GPSRCmdGen resource files.

### CODE REVIEW

File: CommonFiles/Questions.xml [hunk 1]
```diff
@@ -76,4 +52,16 @@
...
     <a>It hasn't been announced yet.</a>
   </question>
-</questions>
\ No newline at end of file
+  <question>
+  <q>What is the origin of the name Canada?</q>
+  <a>The name Canada comes from the Iroquois word Kanata, meaning village or settlement.</a>
+  </question>
+  <question>
+  <q>What is the captial of Canada?</q>
```
Comment (LoyVanBeek) [inline CommonFiles/Questions.xml]: Typo "captial"

File: EEGPSRCmdGen/Resources/eegpsr_cat2i.txt [hunk 2]
```diff
@@ -41,10 +43,10 @@ $task      = $describeI
...
 #
 ##############################################################################
-$describeI = tell me how the person $posture at the {beacon?} looks like
-$describeI = describe the person $posture at the {beacon?} to me
-$describeI = describe the person $posture at the {beacon?} to the person at the {beacon}
-$describeI = describe me the person $posture at the {beacon?}
+$describeI = tell me how the person $posture in the {beacon?} looks like
+$describeI = describe the person $posture in the {beacon?} to me
+$describeI = describe the person $posture in the {beacon?} to the person at the {beacon}
+$describeI = describe me the person $posture in the {beacon?}
```
Comment (LoyVanBeek) [inline EEGPSRCmdGen/Resources/eegpsr_cat2i.txt:49]: "to me"?

**Review by LoyVanBeek — CHANGES_REQUESTED** [review]
Not a native speaker here, but I think I found some minor stuff in this PR.
Thanks for the work :-)

**Comment by justinhart** [comment]
The to me part is correct, actually. But I can fix capital!

File: EEGPSRCmdGen/Resources/eegpsr_cat2i.txt (hunk 2 above)
Comment (justinhart) [reply EEGPSRCmdGen/Resources/eegpsr_cat2i.txt:49]: Fixed!

**Comment by justinhart** [comment]
I've fixed the spelling error. The grammatical error, ending the sentence in "to me" is actually the more common way to use those phrases in English.

File: EEGPSRCmdGen/Resources/eegpsr_cat2e.txt [hunk 3]
```diff
@@ -87,16 +89,17 @@ $fgrefuse  = {void meta: The operator will actively refuse to follow/guide the r
 ##############################################################################
 $peopleR   = $ppl are $peopleDsc
 $peopleDsc = $posture | {gesture}
-$peopleDsc = wearing $outfit
-$peopleDsc = wear $color
+$peopleDsc = wearing $outfitss
+$peopleDsc = wearing $color
 
 $people    = $ppl
 $people    = $posture $ppl
```
Comment (jsimmering) [inline EEGPSRCmdGen/Resources/eegpsr_cat2e.txt:96]: In the eegpsr_cat2.txt this case was changed to $ppl $posture is it on purpose that this wasn't changed here?

File: EEGPSRCmdGen/Resources/eegpsr_cat2.txt [hunk 4]
```diff
@@ -93,14 +96,20 @@ $ppl       = people | children | elders | men | women
...
 $person    = $lpdesc (man | woman | person)
+$person    = (man | woman | person) {gesture}
 $person    = (man | woman | person) $rpdesc
-$lpdesc    = {gesture} | tallest | smallest | oldest | youngest | slimmest | fattest
-$rpdesc    = dressed with $color $outfit
-$rpdesc    = wearing ($apparel | $color $outfit)
+$lpdesc    = tallest | smallest | oldest | youngest | slimmest | fattest
+$rpdesc    = dressed in an $coloran $outfita
+$rpdesc    = dressed in a $colora $outfita
+$rpdesc    = wearing ($apparel | a $colora $outfita | an $coloran $outfitan | ($colora | $coloran) $outfits)
```
Comment (jsimmering) [inline EEGPSRCmdGen/Resources/eegpsr_cat2.txt]: I don't think there is a rule for $outfitan is there?

File: EEGPSRCmdGen/Resources/eegpsr_cat2e.txt (hunk 3 above)
Comment (justinhart) [reply EEGPSRCmdGen/Resources/eegpsr_cat2e.txt:96]: The change to eegpsr_cat2.txt is correct and should have been done here as well. I have fixed this.

File: EEGPSRCmdGen/Resources/eegpsr_cat2.txt (hunk 4 above)
Comment (justinhart) [reply EEGPSRCmdGen/Resources/eegpsr_cat2.txt]: References to $outfitan should be $outfita. This has been fixed.

**Review by kyordhel — APPROVED** [review]

**Comment by kyordhel** [comment]
@LoyVanBeek For a clean merge, could you approve?

**Review by LoyVanBeek — APPROVED** [review]

**Pull request merged**